wander-embedded-sdk/src/components/iframe/wander-iframe.component.ts

```diff
@@ -233,6 +233,7 @@ export class WanderIframe {
     }
 
     this.wrapper.dataset.layout = layoutType;
+    this.backdrop.dataset.layout = layoutType;
```

Comment:
I think it's better to fix this by changing these lines in `wander-iframe.component.ts`:

    shadow.appendChild(backdrop);
    shadow.appendChild(halfImage);
    shadow.appendChild(wrapper);

To:

    shadow.appendChild(wrapper);
    shadow.appendChild(halfImage);
    shadow.appendChild(backdrop);

Visually, everything should look the same (as we use `z-index` for that), but this allows devs to target the backdrop using any of the available attributes/classes on `.iframe-wrapper` like so:

    .iframe-wrapper[<SOME_ATTR>].<SOME_CLASS> + .wrapper { ... }